README.md

```diff
@@ -24,8 +24,10 @@ To get details about the system, read our paper in [SOSP '21](https://doi.org/10
 - `catloop` - TCP Socket Loopback LibOS
 - `catmem` - Shared Memory LibOS
 - `catnap` - Linux Sockets LibOS
+- `catnapw` - Windows Winsock LibOS
 - `catnip` - DPDK LibOS
 - `catpowder` - Linux Raw Sockets
+- `catpowderw` - Windows XDP
```

Comment:
We could club here as well to avoid confusion to new readers:

- `catpowder` - Linux Raw Sockets / Windows XDP